app/src/main/java/hm/orz/chaos114/android/slideviewer/SlideViewerApplication.java

```diff
@@ -19,5 +21,13 @@ public void onCreate() {
 
         AnalyticsManager.initializeAnalyticsTracker(this);
         AnalyticsManager.updateUserProperty();
+
+        if (true) {
+            Integer i = null;
+            i.toString();
+
+            SharedPreferences.Editor editor = PreferenceManager.getDefaultSharedPreferences(this).edit();
+            editor.commit();
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/noboru-i/SlideViewer/blob/223249c3bd60162ca4e93a19147b5437131a2ad7/app/src/main/java/hm/orz/chaos114/android/slideviewer/SlideViewerApplication.java#L30"></span>Consider using <code>apply()</code> instead; <code>commit</code> writes its data to persistent storage immediately, whereas <code>apply</code> will handle it in the background</td>
    </tr>
  </tbody>
</table>